src/localization/bundles/en.json

```diff
@@ -71,7 +71,7 @@
     "id.with_colon": "ID:",
     "common.encryptkey.message": "Please provide encryption key for running playbooks",
     "common.encryptkey.title": "Provide encryption key",
-    "common.encryptkey.description": "The operation you are about to perform requires the encryption key. Please provide the encryption key.",
+    "common.encryptkey.description": "The operation you are about to perform runs a playbook that requires the encryption key. If you provide the wrong encryption key, the playbook will fail and you will need to refresh your browser window in order to re-enter the encryption key.",
```

Comment:
I think this can be written a bit more generally that doesn't put as much blame on the user. Perhaps:

> The operation you are about to perform runs a playbook that requires the encryption key. If the encryption key provided is incorrect the playbook will fail and the browser window will need to be refreshed in order to re-enter the encryption key.

Reply:
@dotNomad  will rewording it. thanks.